src/MatrixTypes_Native.f90

```diff
@@ -496,10 +495,21 @@ SUBROUTINE init_DistributedBandedMatrixParam(matrix,Params)
   CALL validParams%get('MatrixType->n',n)
   CALL validParams%get('MatrixType->m',m)
   CALL validParams%get('MatrixType->MPI_Comm_ID',commID)
-  CALL validParams%get('MatrixType->nnz',nnz)
+  CALL validParams%get('MatrixType->nnz',nnz_int)
   CALL validParams%get('MatrixType->blockSize',blockSize)
   CALL validParams%get('MatrixType->nlocal',nlocal)
-  CALL validParams%clear()
+  nnz=INT(nnz_int,kind=8)
+  IF(nnz_int <= 1) THEN
+    IF (validParams%has("MatrixType->dnnz") .AND. validParams%has("MatrixType->onnz")) THEN
+      nnz=0_SLK
+      CALL validParams%get("MatrixType->onnz",onnz)
+      CALL validParams%get("MatrixType->dnnz",dnnz)
+      DO i=1,SIZE(onnz)
+        nnz=nnz+INT(onnz(i)*blockSize,kind=SLK)
```

Comment:
`kind` to `KIND`

Reply:
Ditto on this one, it's fixed in the newest version